### PR TITLE
fix #1295 : deploy email plugin on RPMs

### LINKED ISSUE #1295: Bug/crowdsec-1.3.1-RC2 : Missing email notification plugin on rpm-based pacakges
The email notification plugin binary isn't installed with the rpm supplied packages.

**Technical Information (please complete the following information):**
 - OS: centos
 - Version: 1.3.1-rc2

### PR DESCRIPTION
**Pull request by buixor**

### CODE REVIEW

File: rpm/SPECS/crowdsec.spec [hunk 1]
```diff
@@ -67,10 +67,12 @@ install -m 644 -D %{SOURCE1} %{buildroot}%{_presetdir}
...
 install -m 551 plugins/notifications/http/notification-http %{buildroot}%{_libdir}/%{name}/plugins/
 install -m 551 plugins/notifications/splunk/notification-splunk %{buildroot}%{_libdir}/%{name}/plugins/
+install -m 551 plugins/notifications/email/notification-email %{buildroot}%{_libdir}/%{name}/plugins/
 
 install -m 644 plugins/notifications/slack/slack.yaml %{buildroot}%{_sysconfdir}/crowdsec/notifications/
 install -m 644 plugins/notifications/http/http.yaml %{buildroot}%{_sysconfdir}/crowdsec/notifications/
 install -m 644 plugins/notifications/splunk/splunk.yaml %{buildroot}%{_sysconfdir}/crowdsec/notifications/
+install -m 644 plugins/notifications/email/email.yaml %{buildroot}%{_sysconfdir}/crowdsec/notifications/
 
 %clean
```
Comment (sabban) [inline rpm/SPECS/crowdsec.spec:77]: It seems like it lacks some information in the %files directive

**Review by sabban — CHANGES_REQUESTED** [review]

**Comment by sabban** [comment]
It seems like it lacks some information in the `%files` directive

File: rpm/SPECS/crowdsec.spec (hunk 1 above)
Comment (buixor) [reply rpm/SPECS/crowdsec.spec:77]: nice catch, fixed

**Review by sabban — APPROVED** [review]
lgtm

**Pull request merged**